### DISCUSSION TITLE
Weekly update 6/15/2022

### DISCUSSION THREAD

**Discussion by kirubeltadesse** [Swarmies/sap-core#42] [General]
# Kirubel

###  How do you feel today?

- Thankful for everything  

### What did you do since the last meeting? 

- I have been reading about the middleware in Django 

### What are you planning to do moving forward? 

- Finishing up the update of the circeCI file.

### Anything hampering your process? 

- Nothing is hampering my progress.

**Comment by dinaolmelak** [comment]
# Dinaol

###  How do you feel today?

- feeling Good. 

### What did you do since the last meeting? 

- Tested the scaffold, and learned about docker containers.

### What are you planning to do moving forward? 

- Get my PR merged with the main branch. As it is important for working on my ticket #39 

### Anything hampering your process? 

- Nothing much. I had allocated time for working on SAP on a daily basis just like I said the previous week.
